evm-adapters/src/sputnik/cheatcodes/cheatcode_handler.rs

```diff
@@ -93,7 +93,7 @@ pub(crate) fn convert_log(log: Log) -> Option<String> {
             format!(
                 "{}: {:?}",
                 inner.key,
-                ethers::utils::parse_units(inner.val, inner.decimals.as_u32()).unwrap()
+                ethers::utils::format_units(inner.val, inner.decimals.as_u32()).unwrap().parse::<f64>().unwrap()
```

Comment:
This conversion does not exactly make sense, what do you want to achieve?

`format_units` takes a number and units and it formats them as a string. The double quotation marks you're seeing are due to the `{:?}` in the `format!`, which indicates that it uses the `Debug` impl of a String. You can change it to `format!("{}: {}")` and they should be gone.

Reply:
Ah, I see! Yes, I was basically trying to remove the double quotation marks. Will change the format and resubmit.